platform/ui/src/components/simpleDialog/SimpleDialog.js

```diff
@@ -21,6 +22,31 @@ class SimpleDialog extends Component {
     rootClass: '',
   };
 
+  static InputDialog = ({ onSubmit, defaultValue, title, label, onClose }) => {
+    const [value, setValue] = useState(defaultValue);
+
+    const onSubmitHandler = () => {
+      onSubmit(value);
+    };
+
+    return (
+      <div className="InputDialog">
+        <SimpleDialog
+          headerTitle={title}
+          onClose={onClose}
+          onConfirm={onSubmitHandler}
+        >
+          <TextInput
+            type="text"
+            value={value}
+            onChange={event => setValue(event.target.value)}
+            label={label}
+          />
+        </SimpleDialog>
+      </div>
+    );
+  };
```

Comment:
If we're updating the `SimpleDialog`, we need to do a quick manual regression test on the relabeling workflow.

Reply:
Looks okay in labeling flow. Resolving.